### PR TITLE
[zh] update /ops/configuration/mesh/app-health-check/index.md

### PR DESCRIPTION
**Pull request by Adityayxt**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by Adityayxt** [comment]
修正了不对的翻译

**Comment by windsonsea** [comment]
Remove `lsync.sh` from your commit please. It's not related.

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ owner: istio/wg-user-experience-maintainers
 test: yes
 ---
 
-[Kubernetes 存活和就绪探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)描述了几种配置存活和就绪探针的方法：
+[Kubernetes 活性和就绪探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)描述了几种配置容器活性和就绪探针的方法：
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:17]:
```suggestion
[Kubernetes 存活和就绪探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)描述了几种配置存活和就绪探针的方法：
```
这是 k8s 的一个术语，原文更好，没必要修改，参考：
ocs/tasks/configure-pod-container/configure-liveness-readiness-pr

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 2]
```diff
@@ -14,14 +14,14 @@ owner: istio/wg-user-experience-maintainers
...
 
 1. [命令](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command)
 1. [HTTP 请求](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request)
 1. [TCP 探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe)
 1. [gRPC 探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)
 
-命令方式无需更改即可工作，但 HTTP 请求和 TCP 探针需要 Istio 更改 Pod 的配置。
+命令方式无需更改即可工作，但 HTTP 请求、 TCP 探针和 gRPC 探针需要 Istio 更改 Pod 的配置。
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:24]:
```suggestion
命令方式无需更改即可工作，但 HTTP 请求、TCP 探针和 gRPC 探针需要 Istio 更改 Pod 的配置。
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 3]
```diff
@@ -31,17 +31,17 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
 所以所有 TCP 端口都显示为开放。kubelet 仅检查某个进程是否正在监听指定的端口，
 因此只要 Sidecar 正在运行，该探针就总会成功。
 
-Istio 通过重写应用程序 `PodSpec` 就绪/存活探针来解决这两个问题，
+Istio 通过重写应用程序 `PodSpec` 就绪/活性探针来解决这两个问题，
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:34]:
```suggestion
Istio 通过重写应用程序 `PodSpec` 就绪/存活探针来解决这两个问题，
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 4]
```diff
@@ -31,17 +31,17 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
...
 以便将探针请求发送到 [Sidecar 代理](/zh/docs/reference/commands/pilot-agent/)。
 对于 HTTP 和 gRPC 请求，Sidecar 代理将请求重定向到应用程序并剥离响应体，仅返回响应代码。
 对于 TCP 探针，Sidecar 代理会在避免流量重定向的同时进行端口检查。
 
 在所有内置的 Istio [配置文件](/zh/docs/setup/additional-setup/config-profiles/)中，
 有问题的探针的重写是默认启用的，但可以如下所述禁用。
 
-## 使用命令方式的存活和就绪探针 {#liveness-and-readiness-probes-using-the-command-approach}
+## 使用命令方式的活性和就绪探针 {#liveness-and-readiness-probes-using-the-command-approach}
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:42]:
```suggestion
## 使用命令方式的存活和就绪探针 {#liveness-and-readiness-probes-using-the-command-approach}
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 5]
```diff
@@ -31,17 +31,17 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
...
 
-Istio 提供了一个[存活示例]({{< github_file >}}/samples/health-check/liveness-command.yaml)来实现这种方式。
+Istio 提供了一个[活性示例]({{< github_file >}}/samples/health-check/liveness-command.yaml)来实现这种方式。
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:44]:
```suggestion
Istio 提供了一个[存活示例]({{< github_file >}}/samples/health-check/liveness-command.yaml)来实现这种方式。
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 6]
```diff
@@ -69,15 +69,15 @@ EOF
 $ kubectl -n istio-io-health apply -f <(istioctl kube-inject -f @samples/health-check/liveness-command.yaml@)
 {{< /text >}}
 
-要确认存活探针是否正常工作，请检查示例 Pod 的状态以验证它是否正在运行。
+要确认活性探针是否正常工作，请检查示例 Pod 的状态以验证它是否正在运行。
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:72]:
```suggestion
要确认存活探针是否正常工作，请检查示例 Pod 的状态以验证它是否正在运行。
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 7]
```diff
@@ -69,15 +69,15 @@ EOF
...
 
 {{< text bash >}}
 $ kubectl -n istio-io-health get pod
 NAME                             READY     STATUS    RESTARTS   AGE
 liveness-6857c8775f-zdv9r        2/2       Running   0           4m
 {{< /text >}}
 
-## 使用 HTTP、TCP 和 gRPC 方式的存活和就绪探针 {#liveness-and-readiness-probes-using-the-http-request-approach}
+## 使用 HTTP、TCP 和 gRPC 方式的活性和就绪探针 {#liveness-and-readiness-probes-using-the-http-request-approach}
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:80]:
```suggestion
## 使用 HTTP、TCP 和 gRPC 方式的存活和就绪探针 {#liveness-and-readiness-probes-using-the-http-request-approach}
```

**Review by windsonsea — COMMENTED** [review]
For k8s glossary, see: https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#glossary

**Comment by zirain** [comment]
please create PR with a readable title

**Comment by windsonsea** [comment]
This file has much changed in en upstream, see #13224. Would you like include it in this PR?

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 8]
```diff
@@ -14,14 +14,15 @@ owner: istio/wg-user-experience-maintainers
...
+[Kubernetes 存活和就绪探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
+描述了几种配置存活和就绪探针的方法：
 
 1. [命令](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command)
 1. [HTTP 请求](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request)
 1. [TCP 探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe)
 1. [gRPC 探针](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)
 
-命令方式无需更改即可工作，但 HTTP 请求和 TCP 探针需要 Istio 更改 Pod 的配置。
+命令方式无需更改即可工作，但 HTTP 请求， TCP 探针和 gRPC 探针需要 Istio 更改 Pod 的配置。
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:25]:
```suggestion
命令方式无需更改即可工作，但 HTTP 请求、TCP 探针和 gRPC 探针需要 Istio 更改 Pod 的配置。
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 9]
```diff
@@ -33,6 +34,74 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
 
 Istio 通过重写应用程序 `PodSpec` 就绪/存活探针来解决这两个问题，
 以便将探针请求发送到 [Sidecar 代理](/zh/docs/reference/commands/pilot-agent/)。
+
+## 存活探针重写示例 {#liveness-probe-rewrite-example}
+
+演示存活探针和就绪探针在应用程序 `PodSpec` 级别如何重写，
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:40]:
```suggestion
为了演示存活探针和就绪探针如何在应用程序 `PodSpec` 级别进行重写，
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 10]
```diff
@@ -33,6 +34,74 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
...
+使用[liveness-http-same-port sample]({{< github_file >}}/samples/health-check/liveness-http-same-port.yaml).
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:41]:
```suggestion
可以使用 [liveness-http-same-port 样例]({{< github_file >}}/samples/health-check/liveness-http-same-port.yaml)。
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 11]
```diff
@@ -33,6 +34,74 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
...
+
+首先为示例创建并标记一个命名空间：
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:43]:
```suggestion
首先为此样例应用创建一个命名空间并打好标签：
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 12]
```diff
@@ -33,6 +34,74 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
...
+
+{{< text bash >}}
+$ kubectl create namespace istio-io-health-rewrite
+$ kubectl label namespace istio-io-health-rewrite istio-injection=enabled
+{{< /text >}}
+
+然后部署示例应用程序：
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:50]:
```suggestion
然后部署样例应用程序：
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 13]
```diff
@@ -33,6 +34,74 @@ TCP 探针检查需要特殊处理，因为 Istio 将所有传入的流量重定
...
+{{< text bash json >}}
+$ kubectl get pod "$LIVENESS_POD" -n istio-io-health-rewrite -o json | jq '.spec.containers[0].livenessProbe.httpGet'
+{
+  "path": "/app-health/liveness-http/livez",
+  "port": 15020,
+  "scheme": "HTTP"
+}
+{{< /text >}}
+
+初始的 `livenessProve` 路径现在被映射到边车容器环境变量 `ISTIO_KUBE_APP_PROBERS` 中的新路径上：
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:95]:
```suggestion
原来的 `livenessProve` 路径现在被映射到 Sidecar 容器环境变量 `ISTIO_KUBE_APP_PROBERS` 中的新路径：
```

File: content/zh/docs/ops/configuration/mesh/app-health-check/index.md [hunk 14]
```diff
@@ -48,7 +117,7 @@ Istio 提供了一个[存活示例]({{< github_file >}}/samples/health-check/liv
 $ kubectl create ns istio-io-health
 {{< /text >}}
 
-要配置 `STRICT` 双向 TLS，请运行：
+要配置 `STRICT` 双向 TLS ，请运行：
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/mesh/app-health-check/index.md:120]:
```suggestion
要配置 `STRICT` 双向 TLS，请运行：
```
Do not leave a space before a comma

**Review by windsonsea — COMMENTED** [review]
Some text shall be improved further.

**Comment by windsonsea** [comment]
Hi @Adityayxt , could you take a look those comments above? So we can move this PR forward.

**Comment by Adityayxt** [comment]
> Hi @Adityayxt , could you take a look those comments above? So we can move this PR forward.

I'll deal with it

**Review by windsonsea — APPROVED** [review]
LGTM now.
Kindly remind: each commit shall have a meaningful message

**Comment by zirain** [comment]
> LGTM now. Kindly remind: each commit shall have a meaningful message

It's OK, istio use `squash` merge mode, so a meaningful title is enough.

**Pull request merged**